run-sidecar

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+CURSORLESS_FORK_REPO="git@github.com:cursorless-everywhere/cursorless.git"
+CURSORLESS_FORK_REV="main"
+CURSORLESS_FORK_NAME="pokey-phillco.cursorless-fork"
+
+CURSORLESS_SIDECAR_REPO="git@github.com:cursorless-everywhere/sidecar.git"
+CURSORLESS_SIDECAR_REV="main"
+CURSORLESS_SIDECAR_NAME="phillco.cursorless-sidecar"
+
+if ! [ -x "$(command -v code)" ]; then
+    echo 'Error: code is not installed.' >&2
+    exit 1
+elif ! [ -x "$(command -v yarn)" ]; then
+    echo 'Error: yarn is not installed.' >&2
+    exit 1
+fi
+
+POSITIONAL_ARGS=()
+
+while test $# -gt 0; do
+    case "$1" in
+        -h | --help)
+            echo "run-sidecar - Provision and run the cursorless sidecar against a vscode extensions directory."
+            echo " "
+            echo "run-sidecar [OPTION]... [EXTENSION_DIR]"
+            echo " "
+            echo "options:"
+            echo "-h, --help                    show brief help"
+            echo "-c, --cursorless-revision     use a specific revision of the cursorless fork"
+            echo "-s, --sidecar-revision        use a specific revision of the cursorless sidecar"
+            exit 0
+            ;;
+        -c | --cursorless-revision)
+            CURSORLESS_FORK_REV="$2"
+            shift
+            shift
+            ;;
+        -s | --sidecar-revision)
+            CURSORLESS_SIDECAR_REV="$2"
+            shift
+            shift
+            ;;
+        *)
+            POSITIONAL_ARGS+=("$1")
+            shift
+            break
+            ;;
+    esac
+done
+
+set -- "${POSITIONAL_ARGS[@]}" # restore positional parameters
+
+EXTENSIONS_DIR="$(realpath $1)"
+shift
+
+CODE=$(which code)
+YARN="yarn --silent "
+FLAGS=" --extensions-dir $EXTENSIONS_DIR"
```

Comment:
I wonder if it's worth experimenting with settings profiles https://github.com/microsoft/vscode-docs/blob/vnext/release-notes/v1_71.md

Reply:
It definitely should be, another option would be to segregate everything by using `--extensions-dir` _and_ `--user-data-dir`. I spent a few minutes futzing around with the profiles feature and couldn't immediately get it working, so I'm going to punt on it for now.